### PR TITLE
Adds Thunderbird

### PR DESCRIPTION
**Pull request by vampirebat74**
## About The Pull Request

Adds the Thunderbird, an original aberration of Firebird. It provides a similar challenge to the Firebird through an inversion of its mechanics; the abnormality deals additional damage after a good result.

- Raises its qliphoth counter while damaging the employee on a good result
- Prioritizes area denial and slowdowns over raw damage
- Creates thralls from nearby corpses

## Why It's Good For The Game

- Management is more focused on combat and maximizing success rates
- No artificial difficulty through teleportation
- Rewards players for maximizing temperance
- Penalizes careless players who don't read the records
- W Corp. sound effects

## Changelog
:cl:
add: Added thunderbird
/:cl:

### CODE REVIEW

File: code/modules/projectiles/projectile/ego_bullets/waw.dm [hunk 1]
```diff
@@ -227,3 +227,65 @@
...
 			M.apply_status_effect(/datum/status_effect/sunder_red)
+
+/datum/status_effect/rendRedArmor
+	id = "rend red armor"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = 30 //3 seconds
+	alert_type = null
+	var/original_armor
+
+/datum/status_effect/rendRedArmor/on_apply()
```
Comment (Coxswain-Navigator) [inline code/modules/projectiles/projectile/ego_bullets/waw.dm:238]: Is rendarmor part of thunderbird or remnants of code from something else?

Comment (Coxswain-Navigator) [reply code/modules/projectiles/projectile/ego_bullets/waw.dm:238]: I have reason to believe that this is deprecated code from exuviae E.G.O.; get rid of it.

File: code/modules/projectiles/guns/ego_gun/waw.dm [hunk 2]
```diff
@@ -315,3 +315,62 @@
...
+/obj/item/gun/ego_gun/warring
+	name = "feather of valor"
+	desc = "A shimmering bow adorned with carved wooden panels. It crackes with arcing electricity."
+	icon_state = "warring"
+	inhand_icon_state = "warring"
+	ammo_type = /obj/item/ammo_casing/caseless/ego_warring
+	weapon_weight = WEAPON_HEAVY
+	fire_delay = 9
+	spread = 0
+	//firing sound 1
```
Comment (Kitsunemitsu) [inline code/modules/projectiles/guns/ego_gun/waw.dm:329]: Please add in a special text explaining the weapon

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Small issues with the EGO weapon. pls fix

**Comment by Kitsunemitsu** [comment]
Squash commits

**Comment by vampirebat74** [comment]
> Small issues with the EGO weapon. pls fix

Is this also concerning the special text?

File: code/modules/mob/living/simple_animal/abnormality/waw/thunder_bird.dm [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+	//Ideally it should only glow in its breached state
+	light_color = LIGHT_COLOR_BLUE
+	light_range = 5
+	light_power = 7
+
+	pixel_x = -8
+	base_pixel_x = -8
+
+	//suppression info
+	maxHealth = 2500
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/abnormality/waw/thunder_bird.dm:27]: Decrease to 1800-2100

File: code/modules/mob/living/simple_animal/abnormality/waw/thunder_bird.dm [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+	health = 2500
+	speed = 2
+	move_to_delay = 3
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/abnormality/waw/thunder_bird.dm:30]: Increase to 4. It has area denial and a dash that knocks down, it doesn't need to be this fast

File: code/modules/mob/living/simple_animal/abnormality/waw/thunder_bird.dm [hunk 5]
```diff
@@ -0,0 +1,366 @@
...
+	var/fire_cooldown_time = 3 SECONDS
+	var/fireball_range = 7
+	var/fire_cooldown
+
+	//Melee stats
+	attack_sound = 'sound/abnormalities/thunderbird/tbird_peck.ogg'
+	stat_attack = HARD_CRIT
+	melee_damage_lower = 10
+	melee_damage_upper = 15
+	rapid_melee = 3
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/abnormality/waw/thunder_bird.dm:68]: Reduce to 2, bit too fast, because again, dash and AOE

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Minor base stat changes. It's too oppressive with all this stuff. Maybe as an aleph

**Review by Kitsunemitsu — APPROVED** [review]
Tested thoroughly on the server.
Code LGTM

**Pull request merged**